resources/shellHelperFunctions.sh

```diff
@@ -225,3 +225,17 @@ write_download_info() {
   echo "  \"size\": \"${FILE_SIZE}\"" >> "$DOWNLOAD_INFO_FILEPATH"
   echo } >> "$DOWNLOAD_INFO_FILEPATH"
 }
+
+# set_version sets the file version on mac/ios projects
```

Comment:
Are you changing the mac build.sh script to use `set_version`?

Reply:
Not for now. Only the two iOS build scripts -- Keyman and FirstVoices Keyboards